content/docs/hello-world.md

```diff
@@ -28,7 +28,7 @@ Bu qo'llanmada biz Reactni quruvchi blocklarini ko'rib chiqamiz: elementlar va k
 
 >Maslahat
 >
->Bu qo'llanma **tushunchalarni qadam ba qadam o'rganishni** afzal ko'radigan odamlar uchun mo'ljallangan. Agar siz amalda o'rganishni xoxlasangiz bizning [amaliy qo'llanma](/tutorial/tutorial.html) bilan tanishing. Bu qo'llanma va darslik bir birini to'ldiradi.
+>Bu qo'llanma **tushunchalarni qadam ba qadam o'rganishni** afzal ko'radigan odamlar uchun mo'ljallangan. Agar siz amalda o'rganishni xoxlasangiz bizning [amaliy o'quv qo'llanma](/tutorial/tutorial.html) bilan tanishing. Bu qo'llanma va amaliy o'quv qo'llanma bir birini to'ldiradi.
```

Comment:
amaliy darslik deganiz yaxshidirov, guide bilan tutorial ni farqi borda

Reply:
https://uz.reactjs.org/docs/hello-world.html shu yerda `Try it on CodePen` ni ham tarjima qilvorish kerak ekan a

Reply:
faqat qattan tarjima qilinishini tushunmadim(